modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/autosize/autosize.es.js

```diff
@@ -60,20 +76,8 @@ class AutoSize {
 	handleInput = (event) => {
 		requestAnimationFrame(() => {
 			const target = event.target;
```

Comment:
I'd just pass `event.target` to `this._resizeInput` instead of defining this variable

Reply:
> I don't see what's changing other than the refactor, I can't find the difference :S

From what I can see, the difference is that we're now calling `this._resizeInput(this.inputElement);` in the constructor to make sure it's properly resized

Reply:
Thanks @julien !

Reply:
no worries :+1: